dev-docs/modules/gdprEnforcement.md

```diff
@@ -49,15 +50,20 @@ A page needs to define configuration rules about how Prebid.js should enforce ea
 {: .alert.alert-warning :}
 **Important Legal Note:** Prebid.org cannot provide legal advice about GDPR or any other governmental regulation. Our aim is to provide a toolkit of functionality that will let publishers configure header bidding as defined by their legal counsel. We will consider feature suggestions, and review any code offered by the community.
 
+{: .alert.alert-info :}
+There are two steps needed to turn on Prebid.js enforcement:
```

Comment:
Can we just say "Follow these step to turn on ..."

Never know if future privacy changes will require more or less steps.

Reply:
I made some edits to address this.